test/integration/suites/get-authorized-entries/03-assert-entities-created.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+SPIFFE_ID="spiffe://example.org/workload"
+
+echo "Checking if entry with SPIFFE ID ${SPIFFE_ID} exists..."
+OUTPUT=$(spire-server entry show -spiffeID "${SPIFFE_ID}" 2>&1)
+
+if echo "$OUTPUT" | grep -q "Error"; then
+    echo "Error: Entry with SPIFFE ID ${SPIFFE_ID} not found."
+    exit 1
+else
+    echo "Entry with SPIFFE ID ${SPIFFE_ID} exists."
+fi
```

Comment:
- this test only ends with making sure that spire-server has the entries, but doesn't test GetAuthorizedEntries
- how is this test intended to be different from, for example, https://github.com/spiffe/spire/blob/main/test/integration/suites/fetch-x509-svids/04-create-registration-entries#L17 where we verify an agent receives its entries? are we planning to call GetAuthorizedEntries ourselves, eliminating spire-agent code from the test equation?